streams/src/main/java/org/apache/kafka/streams/processor/ProcessorContext.java

```diff
@@ -105,8 +111,8 @@ void register(final StateStore store,
 
     /**
      * Schedule a periodic operation for processors. A processor may call this method during
-     * {@link Processor#init(ProcessorContext) initialization} or
-     * {@link Processor#process(Object, Object) processing} to
+     * {@link org.apache.kafka.streams.kstream.ValueTransformer#init(ProcessorContext)  initialization} or
+     * {@link org.apache.kafka.streams.kstream.ValueTransformer#transform(Object)  processing} to
```

Comment:
Why mention `ValueTransformer`? Is this intentional?

Reply:
Yes. `ValueTransformer` (while deprecated itself) is the last class which is still used via `KTAbel#transformValues()` which we did not yet deprecate... (There is ticket for it already: https://issues.apache.org/jira/browse/KAFKA-17178)

So we cannot remove `ValueTransformer` yet, and it make sense to refer to it here IMHO.